docs/getting-started.md

```diff
@@ -139,31 +156,40 @@ $ ../../gradlew run --args="-mode storage -action charge -amount 0 -to merchant1
 $ ../../gradlew run --args="-mode storage -action pay -amount 100 -to merchant1 -from user1"
 ```
 
+## Set up database schema with transaction
+
+To apply transaction, we can just add a key `transaction` and value as `true` in Scalar DB scheme.
+
+```json
+{
+  "emoney.account": {
+    "transaction": true,
+    "partition-key": [
+      "id"
+    ],
+    "clustering-key": [],
+    "columns": {
+      "id": "TEXT",
+      "balance": "INT"
+    },
+    "compaction-strategy": "STCS"
+  }
+}
+```
+
+Before reapplying the schema, please drop the existing namespace first by issuing the following. 
+(Sorry you need to issue implementation specific commands to do this.)
+```
+$ java -jar $SCALARDB_HOME/tools/scalar-schema/scalar-schema.jar --cassandra -h localhost -u <CASSNDRA_USER> -p <CASSANDRA_PASSWORD> -f emoney-storage.json -D
```

Comment:
`-f` is not required for C*.

Reply:
Thank you for your suggestion.
Document updated with the above suggestion.